### PR TITLE
Fix shellcheck errors in ldap-sdk

### PR DESCRIPTION
**Pull request by ckelleyRH**
Follow up changes will address the remaining warnings.

### CODE REVIEW

**Comment by ckelleyRH** [comment]
Sorry about the whitespace wrangling, Eclipse saw the tabs, different numbers of spacing for indentation and appears to have done an unprompted clean up :rofl:

**Review by fmarco76 — APPROVED** [review]
LGTM

File: .github/workflows/code-analysis.yml [hunk 1]
```diff
@@ -118,3 +118,5 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
+        with:
+          severity: warning
```
Comment (edewata) [inline .github/workflows/code-analysis.yml:122]: It looks like there are unresolved warnings. I'm not sure how shellcheck works, did this change cause the test to fail? Should we defer this particular change until all warnings are resolved so at least the test will pass?

**Review by edewata — APPROVED** [review]
LGTM. I have a comment but feel free to merge.

File: .github/workflows/code-analysis.yml (hunk 1 above)
Comment (ckelleyRH) [reply .github/workflows/code-analysis.yml:122]: This PR resolved the error-level severity problems, and this causes failure on warnings - I am setting the same severity level in all repos. If you don't provide a severity level it fails for any detected issue so the test was already failing. I will address the warning-level problems in follow up PRs as there are ~100 and it was going to get too big.

**Comment by ckelleyRH** [comment]
Thanks @edewata @fmarco76 !

File: .github/workflows/code-analysis.yml (hunk 1 above)
Comment (fmarco76) [reply .github/workflows/code-analysis.yml:122]: The majority of the problems in this repository are on the c sdk which is not built, used and maintained (AFAIK). We were discussing to drop in the legacy repository so you can take this in account to decide which script shell fix.

Comment (ckelleyRH) [reply .github/workflows/code-analysis.yml:122]: Good point, in my next update I will disable the check on `c-sdk`.

**Pull request merged**